### PR TITLE
v6.0.0-alpha.11

### PR DESCRIPTION
**Pull request by cherniavskii**
- [x] Compare the last tag with the branch upon which you want to release (`next` for the alpha / beta releases and `master` for the current stable version).
      To do so, use `yarn release:changelog` The options are the following:

  ```bash
  yarn release:changelog
     --githubToken   YOUR_GITHUB_TOKEN (needs "public_repo" permission)
     --lastRelease   The release to compare against (default: the last one)
     --release       The branch to release (default: master)
  ```

  You can also provide the github token by setting `process.env.GITHUB_TOKEN` variable.

  In case of a problem, another method to generate the changelog is available at the end of this page.

- [x] Clean the generated changelog, to match the format of [https://github.com/mui/mui-x/releases](https://github.com/mui/mui-x/releases).
- [x] Update the root `package.json`'s version
- [x] Update the versions of the other `package.json` files and of the dependencies with `yarn release:version`.
- [x] Fix manually the package version in `x-date-pickers/package.json` and `x-date-pickers-pro/package.json`.
- [x] Open PR with changes and wait for review and green CI.
- [ ] Merge PR once CI is green, and it has been approved.

### Release the packages

- [ ] Checkout the last version of the working branch
- [ ] Make sure you have the latest dependencies installed: `yarn`.
- [ ] Build the packages: `yarn release:build`.
- [ ] Release the versions on npm: `yarn release:publish` (you need your 2FA device).
- [ ] Push the newly created tag: `yarn release:tag`.

### Publish the documentation

The documentation must be updated on the `docs-vX` branch (`docs-v4` for `v4.X` releases, `docs-v5` for `v5.X` releases, ...)

- [ ] Push the working branch on the documentation release branch to deploy the documentation with the latest changes.

```sh
git push upstream next:docs-next -f
```

You can follow the deployment process [on the Netlify Dashboard](https://app.netlify.com/sites/material-ui-x/deploys?filter=docs-v5)
Once deployed, it will be accessible at https://material-ui-x.netlify.app/ for the `docs-v5` deployment.

### Publish GitHub release

- [ ] After documentation is deployed, publish a new release on [GitHub releases page](https://github.com/mui/mui-x/releases)

### Announce

- [ ] Follow the instructions in https://mui-org.notion.site/Releases-7490ef9581b4447ebdbf86b13164272d.

### CODE REVIEW

File: packages/x-license-pro/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "6.0.0-alpha.9",
+  "version": "6.0.0-alpha.11",
```
Comment (LukasTy) [inline packages/x-license-pro/package.json:3]: Was there any change impacting `x-license-pro` package and warranting a release?

Comment (LukasTy) [reply packages/x-license-pro/package.json:3]: Ah, it's probably the `mui core` [version bump](https://github.com/mui/mui-x/commit/a493eb2bb2f358cce360dc2d5fdf011f866c52b7)? 🤔 
It is only a bump in patch version. In theory it should not force any users to have different package versions resolved, because of this mismatch...
What do you think?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,117 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.0.0-alpha.11
+
+_Dec 8, 2022_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Add dragging support for `DateRangePicker` (#6763) @LukasTy
```
Comment (flaviendelangle) [inline CHANGELOG.md:12]: The range support is not available on the `DateRangePIcker` component, only on `NextDateRangePicker` and `DateRangeCalendar`.

Maybe rephrase to something like

```suggestion
- 🚀 Add dragging support for the new Date Range Picker (`NextDateRangePicker`) (#6763) @LukasTy
```

To avoid any confusion

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,117 @@
...
+- ✨ Add missing lazy-loading feature to `DataGridPremium` (#7124) @m4theushw
```
Comment (flaviendelangle) [inline CHANGELOG.md:13]: Not knowing what this PR did, I did not understand the highlight.
I thought you developed a new feature that the community thought was missing.
But you ""just"" added a feature already developed and that we forgot to call on the premium component.

Maybe we could phrase it better

File: packages/x-license-pro/package.json (hunk 1 above)
Comment (cherniavskii) [reply packages/x-license-pro/package.json:3]: I assumed that lerna only bumps the version of the changed packages.
For the license package, there was only this change: https://github.com/mui/mui-x/pull/7011/files#diff-09f4b5272f6882385a722c3dd6d78fc3512474df9ebaadcd4713221ff0ce9ae0

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,117 @@
...
+
+### `@mui/x-data-grid@v6.0.0-alpha.11` / `@mui/x-data-grid-pro@v6.0.0-alpha.11` / `@mui/x-data-grid-premium@v6.0.0-alpha.11`
+
+#### Breaking changes
+
+- `filterPanelOperators` translation key was renamed to `filterPanelOperator` (#7062) @MBilalShafi
+
+#### Changes
+
+- [DataGrid] Add `GridCell` change in migration guide (#7087) @MBilalShafi
```
Comment (flaviendelangle) [inline CHANGELOG.md:24]:
```suggestion
- [docs] Add `GridCell` change in migration guide (#7087) @MBilalShafi
```

And move to the doc, unless it has changes on the package itself (in which case we need to update the description of the PR)

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,117 @@
...
+- [DataGrid] Fixes rows not rendering properly after height change (#6892) @MBilalShafi
+- [DataGrid] Remove `Header` slot (#6999) @cherniavskii
```
Comment (flaviendelangle) [inline CHANGELOG.md:26]: We don't document this breaking change ?

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,117 @@
...
+     }}
+   />
+  ```
+
+#### Changes
+
+- [DateRangePicker] Add dragging support to edit range (#6763) @LukasTy
+- [pickers] Fix lost props on Date Range Pickers (#7092) @flaviendelangle
+- [pickers] Fix toolbar on the new range pickers (#6989) @flaviendelangle
+- [pickers] Improve performance of `DayCalendar` (#7066) @flaviendelangle
```
Comment (flaviendelangle) [inline CHANGELOG.md:99]: That one could be a highlight, it's a massive perf improvement.
Maybe just a `- Performance improvements` like we have `- Doc fixes`

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,117 @@
...
+### Docs
+
+- [docs] Fix API page ad space regression (#7051) @oliviertassinari
+- [docs] Update localization doc to use existing locale (#7102) @LukasTy
+
+### Core
+
+- [core] Add codemod to move l10n translation (#7027) @alexfauquette
+- [core] Add notes to remove the legacy pickers internals (#7133) @flaviendelangle
+- [core] Remove internals-fields imports (#7119) @flaviendelangle
```
Comment (flaviendelangle) [inline CHANGELOG.md:112]:
```suggestion
- [core] Remove `internals-fields` imports (#7119) @flaviendelangle
```

File: CHANGELOG.md (hunk 3 above)
Comment (cherniavskii) [reply CHANGELOG.md:13]: Maybe this would be more clear?

```suggestion
- ✨ Fix lazy-loading feature not working in `DataGridPremium` (#7124) @m4theushw
```

File: CHANGELOG.md (hunk 4 above)
Comment (MBilalShafi) [reply CHANGELOG.md:24]: Yes, it's docs only change, sorry about the wrong label 🙈

File: packages/x-codemod/package.json [hunk 8]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-codemod",
-  "version": "6.0.0-alpha.9",
+  "version": "6.0.0-alpha.11",
```
Comment (MBilalShafi) [inline packages/x-codemod/package.json:3]: If we bump to the latest version (i.e. `alpha.11`) for other packages, should we do the same for `@mui/x-license-pro`?

File: CHANGELOG.md [hunk 9]
```diff
@@ -9,20 +9,21 @@ _Dec 8, 2022_
 
 We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
 
-- 🚀 Add dragging support for `DateRangePicker` (#6763) @LukasTy
-- ✨ Add missing lazy-loading feature to `DataGridPremium` (#7124) @m4theushw
+- 🚀 Add dragging support for the new Date Range Picker (`NextDateRangePicker`) (#6763) @LukasTy
+- ⚡️ Improve performance of `DayCalendar` (#7066) @flaviendelangle
```
Comment (flaviendelangle) [inline CHANGELOG.md:13]:
```suggestion
- ⚡️ Improve performance of the `day` view (#7066) @flaviendelangle
```

This component is private, let's use the view which is how devs can enable it :+1:

**Review by flaviendelangle — APPROVED** [review]
Looks great !
Some very nice changes on the picker side this week.

File: packages/x-codemod/package.json (hunk 8 above)
Comment (cherniavskii) [reply packages/x-codemod/package.json:3]: For the `x-license-pro` see https://github.com/mui/mui-x/pull/7144#discussion_r1043527877
